File: Easy/26. Remove Duplicates from Sorted Array/official solution (1).java
// 26. Remove Duplicates from Sorted Array

// Runtime: 0 ms, faster than 100.00% of Java online submissions for Remove Duplicates from Sorted Array.

// Memory Usage: 41.3 MB, less than 21.81% of Java online submissions for Remove Duplicates from Sorted Array.


class Solution {
    // Two Pointers
    public int removeDuplicates(int[] nums) {
        if (nums.length == 0) {
             return 0;
        }

        int len = 0;
        for (int i = 1; i < nums.length; i++) {
            if (nums[i] != nums[len]) {
                len++;
                nums[len] = nums[i];
            }
        }

        return len + 1;
    }
}

File: Easy/977. Squares of a Sorted Array/official solution (1).py
# 977. Squares of a Sorted Array

# Runtime: 204 ms, faster than 83.47% of Python3 online submissions for Squares of a Sorted Array.


class Solution:
    # Sort
    def sortedSquares(self, A: List[int]) -> List[int]:
        return sorted(x*x for x in A)